src/Operation/Watch.php

```diff
@@ -69,7 +70,9 @@ class ChangeStream implements Executable
      *    This is not supported for server versions < 3.2 and will result in an
```

Comment:
Is this necessary if change streams can only be used on server versions >= 3.6?

Reply:
Good catch. I think that we can remove the earlier sentence as well:

> "Note that a "majority" read concern is not compatible with the $out stage"

According to [the spec](https://github.com/mongodb/specifications/blob/master/source/change-streams.rst#helper-method), "majority" is the only supported read concern for change streams in 3.6; however, it'd be easier to not discuss this in our docs because it's subject to change (much like the `2dsphere` index option).